app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: any) -> any:
+    result_runs = {}
+
+    def wrapper(*args: int) -> any:
+
+        if args not in result_runs.keys():
+            run_func = func(*args)
+            result_func = {args: run_func}
+            result_runs.update(result_func)
```

Comment:
too many lines for one simple action:
```suggestion
            result_runs[args] = func(*args)
```